__tests__/integration.test.js

```diff
@@ -83,3 +84,33 @@ describe("getArticleByID", () => {
   });
 });
 
+describe('getArticles', () => {
+  it('should return status 200 and the correct object', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then((res) => {
+      const {articles} = res.body
+
+      articles.forEach((article) => {
+        expect(typeof article.author).toBe('string');
+        expect(typeof article.title).toBe('string');
+        expect(typeof article.article_id).toBe('number');
+        expect(typeof article.topic).toBe('string');
+        expect(typeof article.created_at).toBe('string');
+        expect(typeof article.votes).toBe('number');
+        expect(typeof article.article_img_url).toBe('string');
+        expect(typeof article.comment_count).toBe('string');
+      });
+    })
+  });
+  it('should return the object in the correct order, with correct lengths', () => {
+    return request(app)
+    .get('/api/articles')
+    .then((res) => {
+      const {articles} = res.body
+      expect(articles).toHaveLength(13);
+    })
```

Comment:
Really good testing so far, but another important test for this endpoint is ensuring that the articles are sorted by created_at in desc order. A hint for this is to use the jest-sorted library to help with this.